packages/vats/src/lib-chainStorage.js

```diff
@@ -10,6 +10,19 @@ const { details: X } = assert;
 // ASCII alphanumeric plus underscore.
 const pathSegmentPattern = /^[a-zA-Z0-9_-]{1,100}$/;
 
+/**
+ * @param {string} name
+ */
+export const sanitizePathSegment = name => {
+  const candidate = name.replace(/[- ]/g, '_');
```

Comment:
@turadg I missed this at the time, but two questions:
1. What is the motivation for replacing `-` (which is a valid path segment character)?
2. Shouldn't this rather specific function be defined in a higher layer?

Reply:
Let's discuss in https://github.com/Agoric/agoric-sdk/pull/5926